scripts/dts/extract_dts_includes.py

```diff
@@ -84,7 +84,7 @@ def generate_prop_defines(node_path, prop):
         compatible.extract(node_path, prop, def_label)
     elif 'clocks' in prop:
         clocks.extract(node_path, prop, def_label)
-    elif 'pwms' in prop or 'gpios' in prop:
+    elif 'pwms' in prop or '-gpios' in prop or "gpios" == prop:
```

Comment:
Was that meant to be `prop == "gpios"`? Looks a bit Yoda.

The old code is broken in that `"-gpios" in "foo-gpios-lol"` is true, but it's not a regression.

Reply:
Fixed up.